docs/integrations/tools/llmlingua-prompt-compression-tool.md

```diff
@@ -0,0 +1,42 @@
+# LLMLingua Prompt Compression
+
+## Introduction
+LLMLingua Prompt Compression tool enables you to speed up LLMs' inference and enhance LLM's perceive of key information, compress the prompt with minimal performance loss.
```

Comment:
note the spell check is failed, please update /.cspell.json file

Reply:
updated the md file and change 'LLM' to 'large language model'